examples/disque-job-queue/build.gradle

```diff
@@ -6,8 +6,11 @@ repositories {
     jcenter()
 }
 
+def lombokVersion = '1.18.10'
+
 dependencies {
-    compileOnly 'org.projectlombok:lombok:1.18.10'
+    compileOnly "org.projectlombok:lombok:$lombokVersion"
```

Comment:
If we extract the version as a variable, Dependabot won't be able to bump it.

Even though it would be less DRY to leave the version number in place, I think it ought to actually be more maintainable since keeping the lib up to date should be fully automated.

Reply:
Ah yes, very good :slightly_smiling_face: